src/libsystemd/sd-journal/journal-file.c

```diff
@@ -36,36 +36,36 @@
 #include "sync-util.h"
 #include "xattr-util.h"
 
-#define DEFAULT_DATA_HASH_TABLE_SIZE (2047ULL*sizeof(HashItem))
-#define DEFAULT_FIELD_HASH_TABLE_SIZE (333ULL*sizeof(HashItem))
+#define DEFAULT_DATA_HASH_TABLE_SIZE (2047ULL * sizeof(HashItem))
```

Comment:
Something went wrong here? 1f06ea747b5939a1083c436dd7dae97b37bedee7 doesn't do any reformatting and shouldn't need it either.